### DISCUSSION TITLE
Error with installation script

### DISCUSSION THREAD

**Discussion by bttd** [searx/searx#3303] [Installation issue: Official wiki] [answered]
Hi,

I installing the searxng with built in installation script. But the end of the installation, I got an error

sudo -H ./utils/searx.sh install all

At the end there is an error in logs:

`Enable SearXNG debug mode? [NO/yes] y
WARN:  Do not enable debug in production environments!!
INFO:  try to enable debug mode ...
cd /usr/local/searx/searx-src
sed -i -e "s/debug: false/debug: true/g" "/etc/searxng/settings.yml"
INFO:  restart uWSGI service

Unit searx.service could not be found.
// use CTRL-C to stop monitoring the log
Failed to load settings file: failed to decode base64 data: Invalid base64-encoded string: number of data characters (17) cannot be 1 more than a multiple of 4
  in "/usr/local/searx/searx-src/searx/settings.yml", line 69, column 11
Fri Jul 29 15:02:14 2022 - SIGINT/SIGQUIT received...killing workers...
Fri Jul 29 15:02:14 2022 - gateway "uWSGI http 1" has been buried (pid: 507205)
Fri Jul 29 15:02:14 2022 - Respawned uWSGI worker 2 (new pid: 507506)
Fri Jul 29 15:02:14 2022 - worker 1 buried after 0 seconds
Failed to load settings file: failed to decode base64 data: Invalid base64-encoded string: number of data characters (17) cannot be 1 more than a multiple of 4
  in "/usr/local/searx/searx-src/searx/settings.yml", line 69, column 11
Fri Jul 29 15:02:15 2022 - worker 2 buried after 0 seconds
Fri Jul 29 15:02:15 2022 - goodbye to uWSGI.`

I don't really understand what is the problem here, or how can I solve it. Line 69 in settingy.yml is a comment line.

**Comment by kvch** [comment, marked as answer]
This is the searx repo, not searxng. Please go post your question there.
